lib/content-services/src/lib/document-list/data/share-datatable-adapter.ts

```diff
@@ -267,7 +267,7 @@ export class ShareDataTableAdapter implements DataTableAdapter {
 
                 if (this.sortingMode !== 'server') {
                     // Sort by first sortable or just first column
-                    if (this.columns && this.columns.length > 0) {
+                    if (this.columns?.length > 0) {
```

Comment:
Can we have just like below?
```suggestion
                    if (this.columns?.length) {
```